lib/SPIRV/SPIRVReader.cpp

```diff
@@ -1412,6 +1412,18 @@ Value *SPIRVToLLVM::oclTransConstantPipeStorage(
                             GlobalValue::NotThreadLocal, SPIRAS_Global);
 }
 
+static void replaceOperandWithIntrinsicCallResult(Value *&V) {
```

Comment:
This is minor comment, applying of it is at your discretion.
I think that it is clearer that a function changes its argument if the call looks like `Val = getAnnotationIntrinsicResult(Val);`